doc/help/kdb-meta-get.md

```diff
@@ -42,6 +42,8 @@ This command will return the following values as an exit status:<br>
   Explain what is happening. Prints additional information in case of errors/warnings.
 - `-d`, `--debug`:
   Give debug information. Prints additional debug information in case of errors/warnings.
+- `-N`, `--namespace=NS`:
+  Use the specified namespace in case the provided key does not already have a namespace.
```

Comment:
What if the namespace is provided twice? (I would expect an error from this docu.)

Reply:
For a case like `kdb meta-get -N system user:/key metakey` I opted to use the namespace of the key, since this is how `kdb set` worked in the past, and throwing an error (while still being consistent with `kdb set`) leads to e.g. testcase [check_distribution.sh](https://github.com/ElektraInitiative/libelektra/blob/f910d2345fda33ce269b28f07818ed840b8881de/tests/shell/check_distribution.sh) breaking.

What do you think is the solution?

Reply:
The proper solution probably is to completely get rid of `-N` as we actually want it to be synonymous to add the namespace to the key. I proposed this in #4012

Reply:
The proposal is now fully implemented.